README.md

```diff
@@ -90,6 +111,31 @@ at [our DynaKube Custom Resource example](https://raw.githubusercontent.com/Dyna
 $ kubectl apply -f cr.yaml
 ```
 
+#### Enable Kubernetes API monitoring
+
+To get native Kubernetes metrics, you need to connect the Kubernetes API to Dynatrace.
+
+1.Get the Kubernetes API URL
```

Comment:
An extra space between the number and text would be nice.

Reply:
Added a space